src/sources/tripadvisor/convert.rs

```diff
@@ -82,11 +123,47 @@ pub fn build_poi(
                 .map(ToString::to_string)
         })
         .unwrap_or_else(|| category.clone())
+        .replace(" ", "")
```

Comment:
Maybe we should use an underscore for readability, I don't think this would make existing prefixes (`class_`, `subclass_`, `cuisine_`) confusing?